lib/model/oasis/annotations/AnnotationTarget.js

```diff
@@ -34,6 +37,10 @@ class AnnotationTarget {
     Object.defineProperty(this, "extensions", {
       get: () => extensions,
     });
+
+    Object.defineProperty(this, "model", {
```

Comment:
I think this should go to ComplexType since navigation properties can be defined for EntityType and ComplexType only. Annotation target is parent for all elements that can be annotated (e.g. Property) and not all have to reference model.

Reply:
I think any element in metadata which is "annotated" needs access to the element which is related by annotations.